scripts/hub_bootstrap.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install python3-venv python3-dev python3-setuptools
+
+python3 -m venv hub
+cd hub
+source bin/activate
+pip install --upgrade pip
+python3 -m pip install wheel
+python3 -m pip install homeassistant
+cd ..
+
+export CURRENT_USER=`whoami`
```

Comment:
Most of the unix systems have `$USER` variable available.
So basically you can specify this variable instead of calling `whoami` and making additional variable :)